packages/core/src/components/tag-input/resizableInput.tsx

```diff
@@ -0,0 +1,38 @@
+/* !
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ */
+
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+
+import { Classes, HTMLInputProps } from "../../common";
+
+export type Ref = HTMLInputElement;
+
+export const ResizableInput = forwardRef<Ref, HTMLInputProps>(function ResizableInput(props, ref) {
+    const [content, setContent] = useState("");
+    const [width, setWidth] = useState(0);
+    const span = useRef<HTMLSpanElement>(null);
+
+    useEffect(() => {
+        if (span.current != null) {
+            setWidth(span.current.offsetWidth);
+        }
+    }, [content]);
+
+    const { onChange, ...otherProps } = props;
+
+    const handleInputChange: React.ChangeEventHandler<HTMLInputElement> = evt => {
+        onChange?.(evt);
+        setContent(evt?.target?.value ?? "");
+    };
+
+    return (
+        <span>
+            <span id="hide" ref={span} className={Classes.RESIZABLE_INPUT_SPAN}>
```

Comment:
Not sure why this `id` prop is needed / where it's used, but it's causing aXe a11y failures because all `id`s must be **unique**.

Must either remove `id` prop, or use `uniqueId` for it.

Reply:
PR: https://github.com/palantir/blueprint/pull/5984